webserver/build.gradle

```
@@ -26,5 +26,10 @@ dependencies {
     testImplementation project(':runner-memory')
```

```suggestion
    testImplementation project(':runner-memory')
```

This should be removed to prove it's working ;)